preview_docs.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Copyright 2019 Autodesk, Inc.  All rights reserved.
+#
+# Use of this software is subject to the terms of the Autodesk license agreement
+# provided at the time of installation or download, or which otherwise accompanies
+# this software in either electronic or hard copy form.
+#
+
+# Builds a local preview of the documentation in the parent repository that is
+# including tk-doc-generator as a submodule.
+
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```

Comment:
Could you turn this into a python script? this way it's invocable on Windows as well.

Reply:
It might be more verbose, but it will also be clearer what's going on to people who are less familiar with bash.

Reply:
If you do, please make sure to update the hash-bang to `#!/usr/bin/env python` and put execute permission on the script :)

Reply:
Right, so i did look into this and I think we should hold off. Having a python script so that we can support windows is a valid reason, "avoid bash" i feel is less so. We should be able to write a two line bash script, python is great for lots of things, bash for others. I am not even sure our docker stuff runs on windows right now. I think we should wait with windows support for now and leave that for later. And thus keep the bash stuff for the time being! Hope it makes sense.

Reply:
Ah right, I wasn't thinking right! This all runs through docker so yes, let's leave the bash stuff in. As long as there is an easy way to launch our docker stuff on Windows, that's awesome!